baip-0002.md

```diff
@@ -0,0 +1,62 @@
+    BAIP: 0002
+    Title: Reform of loopholes in feed price mechanism
+    Authors: cn-vote  bitshareschina@163.com
+    Status: Draft
+    Type: Consensus 
+    Created: 2019-10-22
+
+
+# Abstract
+This BAIP defines reforms to the current feed price mechanism vulnerability. The specific program is: The` feed price` is the highest between the `current price` and the ` two-day moving average price` .
+# Motivation
+After the failure of [BSIP42](https://github.com/bitshares/bsips/blob/master/bsip-0042.md), the current feed price mechanism has major loopholes and serious negatives, many cex exchanges use our vulnerability to maliciously short,which seriously damaged the ecological balance, which caused us to suffer many unnecessary losses and hindered the development of the entire ecology.
```

Comment:
> the current feed price mechanism has major loopholes and serious negatives

Nothing is wrong with the price feeding mechanism within the Bitshares core/ui code base. There may be issues with the price feed scripts which provide values for the price feeding mechanism, but you have not defined them - please fully define the 'major loopholes and serious negatives' so we're all on the same page.

> many cex exchanges use our vulnerability to maliciously short

Allegations have only been made against one CEX, which others do you claim are maliciously shorting? Name and shame them.

> which seriously damaged the ecological balance

economic balance instead of ecological balance?

> caused us to suffer many unnecessary losses

The author cn-vote has suffered unnecessary losses? Surely these were fully acknowledged risks when the debt positions were opened?

> hindered the development of the entire ecology.

do you mean economy? Ecology is a branch of biology, not economics 🤔

This has only affected market pegged assets, not UIA/EBA/ABA/PMA so it hasn't affected everything.